doc/howtos/building-images.rst

```diff
@@ -229,3 +229,16 @@ build configuration.
 Alternatively, ``CORE_IMAGE_EXTRA_INSTALL`` can also be used. The
 difference is that this will also affect the initramfs images, which is
 often not intended.
+
+Automatic Cleaning of Old Image Before New One Get Built
+--------------------------------------------------------
+
+Every image built gets copied in the deploy directory. Further, a new build
```

Comment:
On Thu, 2016-03-17 at 09:01 -0700, David Kinder wrote:
> by removing previous images before the new one is built by adding this
> line to your local.conf: RM_OLD_IMAGE = "1"

"after the new one is built" - if building somehow fails, the old one
remains and the symlink still points to it.